Functions/xASL_bids_ValidateNiftiName.m

```diff
@@ -0,0 +1,56 @@
+function xASL_bids_ValidateNiftiName(fileName,perfType)
```

Comment:
I started with the ASL one because I encountered a problem there. Doing the same with the new bids-matlab after the update would be a good idea though! Let's do this in a separate issue? This is generally a new feature/improvement and unrelated to the PR.

Reply:
OK - Michael, can you please make this new issue? Also thinking of aggregating similar things to the same issue.

Reply:
Michael if you created the issue, can you resolve this?